### PR TITLE
alerts/etcd*: Minor adjustments to existing etcd runbooks

### PR DESCRIPTION
**Pull request by lilic**
@RiRa12621 just some minor adjustments to existing runbooks, please take a look thanks.

FYI @hexfusion you might want to adjust the defrag section once your PR gets merged.

### CODE REVIEW

File: alerts/etcMemberDown.md [hunk 1]
```diff
@@ -42,13 +42,31 @@ the master nodes. This is the case when the [machine-config-operator
...
 ```console
 $ etcdctl endpoint health -w table
 ```
+
+## Mitigation
+
+If an upgrade is in progress, the alert may automatically resolve in some time
+when the master node comes up again. If MCO is not working on the master node,
+check the cloud provider to verify if the master node instances are running or not.
+In the case you are running on AWS, the AWS instance retirement might need a manual reboot of the master node.
```
Comment (ravitri) [inline alerts/etcMemberDown.md:71]: I think we can remove AWS specific lines and maintain it to be cloud provider agnostic?

Comment (lilic) [reply alerts/etcMemberDown.md:71]: Wasn't sure as if we know this was an issue in the past, and info might be useful for our end users.

File: alerts/etcMemberDown.md [hunk 2]
```diff
@@ -42,13 +42,31 @@ the master nodes. This is the case when the [machine-config-operator
...
 
-If an upgrade is in progress, the alert may automatically resolve in some time
-when the master node comes up again. If MCO is not working on the master node,
-check the AWS to verify if the master node instances are running or not. AWS
-instance retirement might need a manual reboot of the master node.
+To run `etcdctl` commands, we need to `rsh` into the `etcdctl` container of any
+etcd pod.
+
+```console
+$ oc rsh -c etcdctl -n openshift-etcd $(oc get po -l app=etcd -oname -n openshift-etcd | awk -F"/" 'NR==1{ print $2 }')
```
Comment (lilic) [inline alerts/etcMemberDown.md:51]: Markdown linter is failing here, I don't think we should break the command in two just to keep with line number size.

**Pull request merged**